spec/foodzone/venues/add_branch_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../../../app/foodzone/venues/add_branch'
+
+describe Venues::AddBranch do
+  let(:add_branch) { described_class }
+  let(:venue) { double :venue }
+  let(:branch_attrs) { {} }
+
+  describe '#execute' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.